fe/fe-core/src/main/cup/sql_parser.cup

```diff
@@ -3509,6 +3509,11 @@ select_hints_list ::=
     :}
     ;
 
+sql_no_cache ::=
```

Comment:
```suggestion
opt_sql_no_cache ::=
```

Reply:
ok

Reply:
Doris supports sql_cache and partition_cache. Sql_no_cache grammar means selecting from table with disabled cache. There are no any operation to this grammar. Why not disable sql_cache/partition_cache?